Source/Apps/Tune/Build.cmd

```diff
@@ -8,6 +8,7 @@ set TASMTABS=%TOOLS%\tasm32
 tasm -t180 -g3 -fFF -dWBW tune.asm tune.com tune.lst || exit /b
 tasm -t180 -g3 -fFF -dZX tune.asm tunezx.com tunezx.lst || exit /b
 tasm -t180 -g3 -fFF -dMSX tune.asm tunemsx.com tunemsx.lst || exit /b
+tasm -t180 -g3 -fFF -dWBW vgmplay.asm vgmplay.com vgmplay.lst || exit /b

```

Comment:
I'm not sure that vgmplay belongs in the Tune source directory, but it is fine for now.